support/safe_intrinsics/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Provides a safe wrapper around some x86-64 instructions.
+//!
+//! This is needed because Rust's intrinsics are marked unsafe (despite
+//! these few being completely safe to invoke).
+
+#![no_std]
+// UNSAFETY: Calling a cpu intrinsic.
+#![allow(unsafe_code)]
+
+/// Invokes the cpuid instruction with input values `eax` and `ecx`.
+#[cfg(target_arch = "x86_64")] // xtask-fmt allow-target-arch cpu-intrinsic
```

Comment:
I suggest making this crate opt-out at the xtask fmt lint-level (as we've done with other crates already)

Reply:
How is this accomplished? I am not seeing any linting-specific suppression strings in the xtask fmt code

Reply:
Should be resolved in the latest commit